defaults/main.yml

```diff
@@ -1,25 +1,22 @@
 ---
 puppet_username: 'puppet'
-puppet_etc_dir: '/etc/puppet'
+puppet_etc_dir: '/etc/puppetlabs/puppet'
+puppet_code_dir: '/etc/puppetlabs/code'
 puppet_data_dir: '/var/lib/puppet'
 hiera_credentials_dir: "{{puppet_etc_dir}}/keys"
 puppet_environment: 'production'
 hostname: "{{ inventory_hostname }}"
-puppetlabs_repo_url: 'https://yum.puppetlabs.com/puppetlabs-release-el-7.noarch.rpm'
+puppetlabs_repo_url: 'https://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm'
 passenger_dot_repo_file_url: 'https://oss-binaries.phusionpassenger.com/yum/definitions/el-passenger.repo'
 rpm_packages:
  - git
  - puppet
- - puppet-server
+ - puppetserver
  - rubygems
-ruby_gems:
- - r10k
- - hiera-eyaml
 # workaround: puppet_forge 2.2.7 requires semantic_puppet ~> 1.0 while r10k requires ~> 0.1
 ruby_gems_versions:
   - { gem: 'puppet_forge', version: '2.2.6' }
-  - { gem: 'r10k', version: '' }
-  - { gem: 'hiera-eyaml', version: '' }
```

Comment:
You tested this with a recent reinstall or two, right? 

I was thinking there might have been some weird corner case where the system-wide hiera-eyaml would be needed alongside the puppetmaster installed one. But I'm probably wrong, especially if it worked on a fresh install :)

Reply:
Got worried there because the github comment only showed the red bits!
This has been tested with reinstalls and CentOS 7.7 for both environments.
To be honest I'm not sure what is Right or Wrong. It works and I think it's nice to pin the versions :)

Reply:
Pinning is fine, this was only concerning the exclusion of hiera-eymaml from this list of OS side gems. Which is fine as well if reinstalls have worked.